internal/k8s/ako_init.go

```diff
@@ -387,20 +387,26 @@ func (c *AviController) FullSyncK8s() error {
 	if err != nil {
 		utils.AviLog.Errorf("Unable to retrieve the services during full sync: %s", err)
 		return err
-	} else {
-		for _, svcObj := range svcObjs {
-			isSvcLb := isServiceLBType(svcObj)
-			var key string
-			if isSvcLb {
-				key = utils.L4LBService + "/" + utils.ObjKey(svcObj)
-			} else {
-				if lib.GetAdvancedL4() {
-					continue
-				}
-				key = utils.Service + "/" + utils.ObjKey(svcObj)
+	}
+	for _, svcObj := range svcObjs {
+
+		isSvcLb := isServiceLBType(svcObj)
+		var key string
+		if isSvcLb {
+			svcLabel := utils.ObjKey(svcObj)
+			ns := strings.Split(svcLabel, "/")
+			//Add key if ServiceAPI or advance L4 enabled or NS is valid
```

Comment:
I think this comment can be reworded. Now it seems like add key if( ServiceAPI) or (advance L4 enabled) or (NS is valid).

Reply:
will change it.